.github/workflows/test_pipeline.yml

```diff
@@ -54,4 +50,4 @@ jobs:
           DB_DATABASE: postgres
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: cd ./pipeline && python client.py
+        run: cd ./pipeline && GH_TOKEN=$GH_TOKEN python client.py
```

Comment:
Is this necessary? My tests on #73 are passing without this.

Reply:
No, I will make a new commit and delete it.